buildimage.go

```diff
@@ -32,7 +32,11 @@ func main() {
 		{"go", "get", "-u", "github.com/u-root/u-root"},
 		{"go", "get", "-d", "-v", "-u", "github.com/u-root/NiChrome/..."},
 		{"sudo", "apt", "install", "wireless-tools"},
-		{"go", "run", "github.com/u-root/u-root/.", "-files", "/sbin/iwconfig:bin/iwconfig", "-files", "/sbin/iwlist:bin/iwlist", *uroot, *cmds, *wcmds, *ncmds},
+		{"go", "run", "github.com/u-root/u-root/.", "-files", "bzImage:bzImage", "-files", "/sbin/iwconfig:/bin/iwconfig", "-files", "/sbin/iwlist:/bin/iwlist", *uroot, *cmds, *wcmds, *ncmds},
```

Comment:
this test is failing because you are trying to put the bzImage in the initramfs. However, the dependency test fails because it cannot find the bzImage to put in... Perhaps, passing the kernel with these files in the commit will help pass the test? What do you think, Ron and Gan? @rminnich @GanShun

Reply:
The big problem is that we are putting -files -bzImage:bzImage into this tool when there may not be a bzImage file.

I think maybe we should have a way to pass "extra" files via the command line. That's what I was trying to do yesterday but failed.

Reply:
Done